### ISSUE TITLE
Qwen3 Omni bug

### ISSUE THREAD

**Issue by VietDunghacker** [modelscope/ms-swift#9757]
### Checklist / 检查清单

- [x] I have searched existing issues, and this is a new bug report. / 我已经搜索过现有的 issues，确认这是一个新的 bug report。

### Bug Description / Bug 描述

Line 1674 of https://github.com/modelscope/ms-swift/blob/main/swift/model/models/qwen.py
                loss += self.config.router_aux_loss_coef * aux_loss.to(
                    loss.device)  # make sure to reside in the same device
Should be self.config.text_config instead.

### How to Reproduce / 如何复现

Not necessary.
